tests/js/spec/views/organizationIntegrations/integrationRow.spec.jsx

```diff
@@ -201,9 +201,6 @@ describe('IntegrationRow', function() {
       );
       expect(wrapper.find('PluginIcon').props().pluginId).toEqual('amazon-sqs');
       expect(wrapper.find('IntegrationName').props().children).toEqual('Amazon SQS');
-      expect(wrapper.find('IntegrationName').props().to).toEqual(
```

Comment:
@NisanthanNanthakumar what's wrong with this spec?

Reply:
@scefali https://github.com/getsentry/sentry/pull/17065#discussion_r381533499